internal/hud/prompt/prompt.go

```diff
@@ -208,9 +208,22 @@ func (p *TerminalPrompt) OnChange(ctx context.Context, st store.RStore) {
 					st.Dispatch(SwitchTerminalModeAction{Mode: store.TerminalModeStream})
 					msg.stopCh <- true
 
+				case 't':
+					warningMsg := "We're removing the legacy terminal mode key shortcut from the prompt.\n" +
```

Comment:
let's remove the warning message and the Sleep too, since this isn't going away

Reply:
Oooh another neat Go syntax trick! Will make those changes!